bp_t5_texts.npy haven't been uploaded because of big size, you can generate it by running data_process.py.
